vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/rules/TestDBRule.java

```diff
@@ -110,21 +109,18 @@ private void restoreDump(URL dumpLocation, String databaseName) throws IOExcepti
         logger.info("mongorestore exit value: " + exec.exitValue());
     }
 
-    public VariantMongoDBAdaptor getVariantMongoDBAdaptor(String dbName)
-            throws IOException, IllegalOpenCGACredentialsException {
+    public VariantWithSamplesAndAnnotationsService getVariantMongoDBAdaptor(String dbName)
+            throws IOException {
         Properties evaTestProperties = new Properties();
         evaTestProperties.load(this.getClass().getResourceAsStream("/evaTest.properties"));
 
         String host = evaTestProperties.getProperty("eva.mongo.host");
         String server = host.split(":")[0];
         int port = Integer.parseInt(host.split(":")[1]);
         String randomDatabaseName = databaseMapping.get(dbName);
-        MongoCredentials credentials = new MongoCredentials(server, port, randomDatabaseName, null, null);
-        VariantMongoDBAdaptor variantDBAdaptor = new VariantMongoDBAdaptor(credentials,
-                evaTestProperties.getProperty(
-                        "eva.mongo.collections.variants"),
-                evaTestProperties.getProperty(
-                        "eva.mongo.collections.files"));
+//        MongoCredentials credentials = new MongoCredentials(server, port, randomDatabaseName, null, null);
+        //todo
+        VariantWithSamplesAndAnnotationsService variantDBAdaptor = null;
```

Comment:
Fix this so all the tests that depends on this Rule work again. Some Spring configuration to build the Service may be needed.